var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -17,68 +17,160 @@ class UfsWeatherModel(CMakePackage):
     url = "https://github.com/ufs-community/ufs-weather-model/archive/refs/tags/ufs-v1.1.0.tar.gz"
     git = "https://github.com/ufs-community/ufs-weather-model.git"
 
-    maintainers("t-brown")
+    maintainers("AlexanderRichert-NOAA")
 
-    version(
-        "2.0.0",
-        tag="ufs-v2.0.0",
-        commit="e3cb92f1cd8941c019ee5ef7da5c9aef67d55cf8",
-        submodules=True,
-    )
-    version(
-        "1.1.0",
-        tag="ufs-v1.1.0",
-        commit="5bea16b6d41d810dc2e45cba0fa3841f45ea7c7a",
-        submodules=True,
-    )
+    version("develop", branch="develop", submodules=True)
+    version("2.0.0", tag="ufs-v2.0.0", submodules=True)
+    version("1.1.0", tag="ufs-v1.1.0", submodules=True)
```

Comment:
You should keep the commit information around on the tags -- that allows Spack to create binary caches for the package and actually know whether two versions are the same.